ngi_reports/common/ign_sample_report.py

```diff
@@ -35,7 +36,7 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.report_dir = os.path.join('delivery', 'reports')
         self.info['support_email'] = config.get('ngi_reports', 'support_email')
         self.info['date'] = datetime.today().strftime('%Y-%m-%d')
-        self.project['sequencing_centre'] = 'NGI {}'.format(self.ngi_node.title())
+        self.project['sequencing_centre'] = 'NGI {}'.format(config.get('ngi_reports', 'ngi_node'))
```

Comment:
Nope, can't do this sorry. Bit of a long story.. The config would be nicest (this is how I originally intended it to be used), but both Uppsala and Stockholm run reports from the same `funk` account, which has to share the same config file.

Instead, the package now looks for a Piper setup XML file and gets the node from the `NGI-S` or `NGI-U` field. If it can't find this, it falls over and the node has to be specified with `--node`.

This value is used in the starting function to choose which python file to import (`ngi_reports/stockholm/ign_sample_report.py` or `ngi_reports/uppsala/ign_sample_report.py`). These files then import this common file.

So, the node should be [specified](https://github.com/NationalGenomicsInfrastructure/ngi_reports/blob/master/ngi_reports/stockholm/ign_sample_report.py#L15-L16) in the scripts that are imported so that the value can then be used here.

Apologies, this is a relatively recent change and a bit of a messy one just to get it to work. Any more elegant solutions are welcome!